Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -279,32 +276,28 @@ public void onSliderClick(SliderSetting item, int position)
     mClickedPosition = position;
     mSeekbarMinValue = item.getMin();
     mSeekbarProgress = item.getSelectedValue(getSettings());
-    AlertDialog.Builder builder = new AlertDialog.Builder(mView.getActivity());
 
     LayoutInflater inflater = LayoutInflater.from(mView.getActivity());
-    View view = inflater.inflate(R.layout.dialog_seekbar, null);
+    View view = inflater.inflate(R.layout.dialog_slider, null);
 
-    builder.setTitle(item.getName());
-    builder.setView(view);
-    builder.setPositiveButton(R.string.ok, this);
-    mDialog = builder.show();
+    mDialog = new MaterialAlertDialogBuilder(mView.getActivity())
+            .setTitle(item.getName())
+            .setView(view)
+            .setPositiveButton(R.string.ok, this)
+            .show();
 
     mTextSliderValue = view.findViewById(R.id.text_value);
     mTextSliderValue.setText(String.valueOf(mSeekbarProgress));
 
     TextView units = view.findViewById(R.id.text_units);
     units.setText(item.getUnits());
 
-    SeekBar seekbar = view.findViewById(R.id.seekbar);
-
-    // TODO: Once we require API 26, uncomment this line and remove the mSeekbarMinValue variable
-    //seekbar.setMin(item.getMin());
-
-    seekbar.setMax(item.getMax() - mSeekbarMinValue);
-    seekbar.setProgress(mSeekbarProgress - mSeekbarMinValue);
-    seekbar.setKeyProgressIncrement(5);
-
-    seekbar.setOnSeekBarChangeListener(this);
+    Slider slider = view.findViewById(R.id.slider);
+    slider.setValueFrom(item.getMin());
```

Comment:
Since you're calling setValueFrom, the value returned by getValue will vary between the value passed to setValueFrom and the value passed to setValueTo, right? If so, you should remove mSeekbarMinValue and all math associated with it, as mentioned in the comment that you've deleted.